pkg/apis/servicecatalog/v1alpha1/types.go

```diff
@@ -51,7 +51,7 @@ type BrokerSpec struct {
 
 	// AuthInfo contains the data that the service catalog should use to authenticate
 	// with the Broker.
-	AuthInfo *BrokerAuthInfo `json:"authInfo,omitempty"`
```

Comment:
This means that the walkthrough is going to involve creating a secret and adding authinfo to the Broker resource - so those need to change in this PR.

@duglin actually has a PR that makes auth optional - are we sure we need this?